src/app/debug/table/table.component.html

```diff
@@ -68,16 +68,20 @@
           <th class="table-row-checkbox header-padding">
             <input data-cy-debug="selectAll" type="checkbox" class="vertical-middle" [checked]="allRowsSelected" (click)="selectAllRows()" title="Select all rows" [style.width]="getCheckBoxSize()" [style.height]="getCheckBoxSize()">
           </th>
-          <th class="text-center header-cell header-padding" *ngFor="let header of viewSettings.currentView.metadataLabels; let i = index"
-              mat-sort-header="{{i}}" [title]="header">{{getShortenedTableHeaderNames(header)}}</th>
+          <th class="text-center table-id-cell header-cell header-padding"
+              *ngFor="let header of viewSettings.currentView.metadataLabels; let i = index"
+              mat-sort-header="{{i}}"
```

Comment:
Can't you do something like this? unless this doesn't work I'd rather see a non interpolated binding
```suggestion
              [mat-sort-header]="i"
```

Reply:
It doesnt accept an integer like that, it only accepts a string, which is why its implented like this.

Reply:
Hm alright, you could've done `i+''` to make it a string

Reply:
Is there a notable difference between: `i+''` and `{{i}}` or just personal preference?

Reply:
`{{i}}` interpolates it into the template and then binds it while `i+''` converts it to string and immediately binds it to the property without needing interpolation

Reply:
Yeah that does sound better, i'll correct it.

Reply:
This does seem to be allowed:
```suggestion
              [attr.mat-sort-header]="i"
```